src/common/git/with-git-hooks.ts

```diff
@@ -0,0 +1,15 @@
+export interface WithGitHooks {
```

Comment:
@quesabe let’s use `type` if that’s possible here.

Reply:
No, JSII noes not let it and fails on checking other interfaces that extend this one. And in case of replacing all the interfaces with type alias, we would eventually face the following error: `JSII1001: Non-primitive types without a symbol cannot be processed.`

Reply:
Got it, thanks.